### DISCUSSION TITLE
Blob images url

### DISCUSSION THREAD

**Discussion by simonTomSt** [vercel/next.js#24175] [Help] [answered]
Is there some solution to use blob in next.js ? Previously I used img and blob URL worked well, but now when I'm trying to use Image component blob url doesn't work. 

Itried to add blob to domains :
![image](https://user-images.githubusercontent.com/64553739/115115240-942e4000-9f93-11eb-9e7e-346a76496361.png)

but i got error form sever because next Image generate that request URL:
![image](https://user-images.githubusercontent.com/64553739/115115271-d22b6400-9f93-11eb-8018-72ece0778a19.png)

instead of :  
 
![image](https://user-images.githubusercontent.com/64553739/115115550-66e29180-9f95-11eb-9d0b-39ec1f0d967c.png)

**Comment by Bexy-Lyn** [comment]
Hey, I am having the same issue. 
After an image upload I want to display the images for preview before the user saves them. So I create an preview url with "URL.createObjectUrl()", which works with the hmtl img, but the Image component throws an error that I have to configure the domain im my config file :/
I would be grateful for any help.

**Reply by akellbl4** [reply]
Hi @Bexy-Lyn,
As I mentioned below for your case you need to use just `<img>` for an image in a blob and `<Image/>` when you have a link to an image.

**Comment by akellbl4** [comment]
Hey!
If I understood correctly, you need to use just `<img>` for blob URLs.
The key mechanism of `next/image` is getting an image by  URL, resize and optimize, and return it. If you provide blob as URL it trying to go by link and there is no link.

If there is not your case, could you please provide more info like: what error do you get? what in the body of response?

Also, It's okay when Next.js makes it safe. Just because Next.js put your URL as a part of final URL it does `encodeURI()` and it works just fine.

**Comment by stefanprobst** [comment]
fyi, blob urls should now be handled correctly in version `11.1.1-canary.7`.

**Comment by simonTomSt** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/64553739/129452147-09df20cb-3329-492e-a46f-732f3c6c36bb.png)
